mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_marc_authority_job_profile.sql

```diff
@@ -22,3 +22,144 @@ INSERT INTO ${myuniversity}_${mymodule}.job_profiles (id, jsonb) values
     "updatedByUserId": "00000000-0000-0000-0000-000000000000"
   }
 }') ON CONFLICT DO NOTHING;
+
+INSERT INTO ${myuniversity}_${mymodule}.action_profiles (id, jsonb) values
```

Comment:
Can we delete action action profile? Or we should restrict this possibility, the same as for derive marc records.